exporter/prometheus/prometheus.go

```diff
@@ -270,10 +271,10 @@ func tagsToLabels(tags []tag.Tag) []string {
 
 func newCollector(opts Options, registrar *prometheus.Registry) *collector {
 	return &collector{
+		registerOnce:    sync.Once{},
```

Comment:
You can remove this line.

Reply:
Oops, I left that in by mistake, thanks, I'll remove it.